src/api/localResolvers/activeLoan.js

```diff
@@ -53,7 +54,8 @@ export default () => {
 						}
 					};
 
-					context.cache.writeData({
+					context.cache.writeQuery({
+						query,
```

Comment:
Why did the query need to be imported here and in `addToBasketInterstitial.js`?

Reply:
`writeData` did not require a query; it wrote the data directly to the cache object however it was provided. With Apollo 3 they have added some additional safety checks, including getting rid of the `writeData` function entirely. That leaves using `writeQuery`, which requires providing the query shape that the data is for which prevents writing data in a shape that is incompatible with the schema.